client/.vscode/tasks.json

```diff
@@ -0,0 +1,30 @@
+// Available variables which can be used inside of strings.
```

Comment:
This was added to more closely follow the Microsoft example app. They use a preLaunchTask run the compile-watch command before the debugger begins.
I had the changes that lined the extension up more closely to their example in one commit originally, but they were all squashed with your request. Do you want me to submit the changes that line up the app to the example as a separate PR?